lib/src/group_changes.dart

```diff
@@ -212,6 +240,7 @@ String prelude() => '''
   td.nopad      {padding: 0px;}
   td.nomatch    {background-color: Salmon;}
   td.match      {background-color: PaleGreen;}
+  td.newFailure {background-color: Pink;}
```

Comment:
This is an inconsistent naming convention.

Reply:
Changed to td.newfailure